webpack.config.js

```diff
@@ -12,15 +12,12 @@ module.exports = {
   devtool: '#cheap-module-eval-source-map',
 
   module: {
```

Comment:
As of Webpack 4, `mode` is required to be specified in options:

>You have to choose (`mode` or `--mode`) between two modes now: production or development

https://github.com/webpack/webpack/releases/tag/v4.0.0

Warning in output:

```
WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
```

Reply:
If production is always what we want, we can at least suppress the warning by assigning the option value:

```suggestion
  module: {
    mode: 'production',
```

Optionally as a separate task, it might be nice to support running in development mode for non-minified output debugging.